lib/Dancer2.pm

```diff
@@ -115,102 +115,1022 @@ __END__
 
 =encoding UTF-8
 
+=head1 SYNOPSIS
+
+    get '/' => sub { 'Hello, world!' };
+
 =head1 DESCRIPTION
 
-Dancer2 is the new generation of L<Dancer>, the lightweight web-framework for
-Perl. Dancer2 is a complete rewrite based on L<Moo>.
+Dancer2 is a lightweight micro web framework for Perl. It is the new
+generation of L<Dancer> and replaces L<Dancer>.
+
+If you are converting a L<Dancer> application to L<Dancer2>, the
+L<Dancer2::Manual::Migration> document covers the changes you might
```

Comment:
I'm not sure every app actually needs updating. I don't want to make an assumption about it.